### DISCUSSION TITLE
An easier way to update the nightly in RPI4?

### DISCUSSION THREAD

**Discussion by ghost** [FreeTubeApp/FreeTube#3538] [Q&A] [answered]
Hi. I am using this app almost daily on a RPI4. However, updating it is really a hassle. Removing the older version and downloading and installing the latest version are all seperate jobs to perform. Is there any easier way to update the app to the latest nightly without removing or downloading. For example, is there a repository that i can add into the list of repos under RPI OS so that when i type "sudo apt update", it is included in the upgradable list? Currently i am using the "deb" version of the app and it is perfectly compatible with RPI OS.

PS: I realized that it is also possible to install the SnapStore under RPI OS. However, the snap store only has the "stable" releases available. What i want is to get the latest "nightly" in an easier manner and perform the update with a single click/command.

**Comment by absidue** [comment, marked as answer]
Currently the only official way to get the nightlies is to download them from GitHub Actions.
For the stable releases the only official to get them are the GitHub releases (the website uses the links in the GitHub releases) and flatpak.

You might be able to find unofficial distributions of the nightlies, that would be installable with a single command, however please keep in mind that unofficial releases may have issues that the official ones don't.
